tree-construction/tests11.dat

```diff
@@ -272,6 +272,62 @@
 |       zoomandpan=""
 
 #data
+<!DOCTYPE html><body><svg contentScriptType='' contentStyleType=''externalResourcesRequired='' filterRes=''></svg>
```

Comment:
Is the lack of space between attributes intentional? (If so, should be in \#errors.)